lib/sprockets/commoner/processor.rb

```diff
@@ -104,10 +104,18 @@ def call(input)
           @dependencies << "commoner-environment-variable:#{env}"
         end
 
+        map = nil
+        if Commoner.sprockets4?
+          map = result['map']
+          map = Sprockets::SourceMapUtils.decode_vlq_mappings(map['mappings'], sources: map['sources'], names: map['names'])
+          map = Sprockets::SourceMapUtils.combine_source_maps(input[:metadata][:map], map)
+        end
+
         {
           data: result['code'],
           dependencies: @dependencies,
           required: Set.new(@required),
+          map: map,
```

Comment:
I believe `bundle.rb` also needs to properly update the sourcemap after the header, prelude, and footer are added